aether-producer/producer/__init__.py

```diff
@@ -302,6 +381,16 @@ class TopicManager(object):
         AND s.name = {schema_name}
         LIMIT 1; '''
 
+    # Count how many unique (controlled by kernel) messages should currently be in this topic
+    COUNT_STR = '''
+            SELECT
+                count(*)
+            FROM kernel_entity e
+            inner join kernel_projectschema ps on e.projectschema_id = ps.id
+            inner join kernel_schema s on ps.schema_id = s.id
```

Comment:
we included the schema value within the entity model so you can skip one join
```sql
FROM kernel_entity e
INNER JOIN kernel_schema s ON e.schema_id = s.id
```

Reply:
I think we still need it. A topic in Kafka should reflect all entities that share a project schema. Several project schemas sharing the same schema could generate multiple topics.